models/review.model.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const reviewSchema = new Schema(
+    {
+        user: {
+            type: String,
+            required: true,
+            ref: 'User',
+        },
+        book: {
+            type: String,
+            required: true,
+            ref: 'Book',
+        },
```

Comment:
@saggy2001 Here, as you are not including the whole user and book object but only their id, so you must change the user and book property's name to  `user_id` and `book_id` respectively.

Reply:
@saggy2001 One more thing, are you using user_id in the review or username?

Reply:
I am thinking of populating the review with the required information like the name of the user so we have to use user_id

Reply:
No, you must always use user_id for this because users can have the same name. You can also populate using user_id, isn't it?

Reply:
I am telling the same thing that We use user_id to populate the information like name of user

Reply:
Okay, so you did use user_id in this review model right?

Reply:
Yes

Reply:
I am not changing the properties name because we can have a myreviews endpoint for the user in the future in which we populate the book property to populate all the books reviewed by user and if we change now then at that time we again have to change the name

Reply:
@Codehackerone I've done some changes for populating user information, please review those changes

Reply:
@saggy2001 I think you should change that to user_id and book_id as the review model should contain their id. And we can always populate by user_id. As id is a sure-shot way of identifying the user.

Reply:
Okk👍

Reply:
Hey @Codehackerone I've done the changes in the property names and
here is the postman doc with new responses - [https://documenter.getpostman.com/view/15301567/TzXzDHC8](https://documenter.getpostman.com/view/15301567/TzXzDHC8)

Reply:
Cool!